pom.xml

```diff
@@ -20,6 +20,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <junit.version>4.13.2</junit.version>
         <junit.jupiter.version>5.3.2</junit.jupiter.version>
+        <snappy.java.version>1.1.10.1</snappy.java.version>
```

Comment:
we need to update it to 1.1.10.4

Reply:
Thanks @arihant-confluent 
Updated the version